INN/umbrella-ghn
# Redo the post sharing buttons

## Changes

This pull request makes the following changes:

- adds Largo-style sharing buttons to the top of posts: facebook, twitter, email
- removes the old, nonfunctional `sharrre` sharing buttons from the top of posts
- copies the Goliath theme JS into this theme to remove the sharrre dependency
- dequeues the sharrre script and styles

![Screen Shot 2020-05-05 at 11 45 44](https://user-images.githubusercontent.com/1754187/81086408-7375f300-8ec6-11ea-90c7-c95804ffb919.png)


## Why



For #9 and #GTN

## Testing/Questions

Features that this PR affects:

- JS in the theme, broadly
- articles


Questions that need to be answered before merging:

- [ ] does removing sharrre break anything?
- [x] do we have any indication that GHN is using Co-Authors Plus? I didn't see any, but I kept the conditions for its support in place.
- [x] what's the right word for the Facebook button? "Share"?

Steps to test this PR:

1. /wp-admin/admin.php?page=goliath-admin : uncheck "show share counter below post"; check "show share buttons above post image"
2. view an article such as `/2020/04/reusable-protective-gear-rescue/`
3. test the buttons:
    - Facebook button should open a Facebook share intent window
    - Tweet button should open a Twitter share intent window, with the "via" parameter of either 1) the author's twitter handle if that's set in the author profile or 2) the sitewide twitter handle that's used for the twitter button in the header
    - Email button should open your preferred email client with an excerpt and link included
4. test the alternate case of the Tweet button by setting/unsetting the post author's twitter handle. For the post given above, the author is `wp-admin/user-edit.php?user_id=140`
